File: generated-samples/csharp/chat-with-vision-reasoning/Sample.cs
using System.ClientModel;
using Azure.AI.OpenAI;
using OpenAI.Chat;

const string endpoint = "<%= endpoint %>";;
const string apiKey = "<some-api-key>";;
const string deploymentName = "<some-deployment-name>";;

AzureOpenAIClient client = new(new Uri(endpoint), new ApiKeyCredential(apiKey));

ChatClient chatClient = client.GetChatClient(deploymentName);

ChatCompletion completion = await chatClient.CompleteChatAsync(
    [
        new UserChatMessage("What is the capital of France?"),
    ],
    new ChatCompletionOptions
    {
      MaxOutputTokenCount = 2000
afasfasfd
    });

Console.WriteLine(completion.Content[0].Text);

File: generated-samples/python/embeddings-keyAuth/sample.py
from openai import OpenAI

endpoint = "<%= endpoint %>"
deployment_name = "<some-deployment-name>"
api_key = "<%= apiKey %>"

client = OpenAI(
    base_url = endpoint,
    api_key = api_key,
)

response = client.embeddings.create(
    input = "How do I use Python in VS Code?",
    model = deployment_name
)
print(response.data[0].embedding)